resources/compass/charts/director/templates/deployment.yaml

```diff
@@ -70,7 +70,7 @@ spec:
             - name: APP_ONE_TIME_TOKEN_URL
               value: "http://compass-connector-internal.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.global.connector.graphql.internal.port }}/graphql"
             - name: APP_CONNECTOR_URL
-              value: "http://compass-connector.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.global.connector.graphql.external.port }}/graphql"
+              value: "https://{{ .Values.global.gateway.mtls.host }}.{{ .Values.global.ingress.domainName }}/connector/graphql"
```

Comment:
Please set it to `tls` not `mtls` host :)

Reply:
Done!